experimental/auth-logic/wrappers/provenance_wrapper_test.go

```diff
@@ -46,13 +35,17 @@ expected_hash("oak_functions_loader::Binary", sha256:15dc16c42a4ac9ed77f337a4a30
 	// the directory structure of the WORKSPACE, so we need to change
 	// to the root directory of the transparent-release project to
 	// be able to read the SLSA files.
-	os.Chdir("../../")
+	os.Chdir("../../../")
+
+	testProvenance := ProvenanceWrapper{FilePath: schemaExamplePath}
+
+  appName, err := GetAppNameFromProvenance(schemaExamplePath)
```

Comment:
This indentation seems based on spaces instead of tabs (and incorrectly aligned)